src/components/FiltersBundle/types.ts

```diff
@@ -18,6 +18,13 @@ export interface ResetFilter {
 
 export interface SearchFilter {
   onChange: (value: string) => void;
+  value?: string;
+  inputRef?: React.RefObject<HTMLInputElement>;
+  style?: {
```

Comment:
we should name this property with other names to avoid confusion with the HTML attribute `style` as it is not behaving as the `style` property and it has specific values. Maybe `widthStyles` could work better?

Reply:
done